pkg/client/observatorium/observability_config.go

```diff
@@ -94,6 +135,52 @@ func (c *ObservabilityCloudWatchLoggingConfig) validate() error {
 	return nil
 }
 
+func validateExactlyOneOIDCTypeProvided(sl validator.StructLevel) {
+	config := sl.Current().Interface().(DataPlaneObservabilityRemoteWriteAuthTypeOIDC)
```

Comment:
This validation should only be performed if authentication is of the OIDC type

Reply:
This is related to https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1647/files#r1149416320